lesson07/assignment/vue-hwk/vue-project/src/components/ImgComp.vue

```diff
@@ -0,0 +1,65 @@
+<script>
+
+
+export
+default {
+ name:'ImgComp',
+ props:{
+// I had the wrong syntax here with the [] instead of {}.  I also validated the types as String
+   imagepath:
```

Comment:
I would have these on the same line for legibility.

`imagepath: String,
alttext: String`